cvat/apps/engine/serializers.py

```diff
@@ -1204,8 +1204,17 @@ def update(self, instance, validated_data):
         _update_related_storages(instance, validated_data)
 
         instance.save()
+
+        if 'label_set' in validated_data and not instance.project_id:
+            self.update_child_objects_on_labels_update(instance)
+
         return instance
 
+    def update_child_objects_on_labels_update(self, instance: models.Task):
+        models.Job.objects.filter(
+            updated_date__lt=instance.updated_date
```

Comment:
I think you forgot the filter by `task_id`.

Reply:
Yes, thanks. Fixed